app/routes/users/components/Penalties.tsx

```diff
@@ -53,6 +53,8 @@ function Penalties({
                     title="Slett prikk"
                     message="Er du sikker på at du vil slette denne prikken?"
                     onConfirm={() => deletePenalty(penalty.id)}
+                    closeOnConfirm={true}
+                    closeOnCancel={true}
```

Comment:
```suggestion
                    closeOnConfirm
```

`closeOnCancel` is true by default.